wurst/_handles/Group.wurst

```diff
@@ -22,6 +23,13 @@ public function group.enumUnitsInRect(rect rec, boolexpr filter)
 public function group.enumUnitsSelected(player p, boolexpr filter)
 	GroupEnumUnitsSelected(this, p, filter)
 
+public function group.enumUnitsOfPlayer(player p, boolexpr filter)
+	GroupEnumUnitsOfPlayer(this, p, filter)
+
+public function group.enumUnitsAll(boolexpr filter)
+	for i = 0 to bj_MAX_PLAYER_SLOTS - 1
```

Comment:
What value does `bj_MAX_PLAYER_SLOTS` take on before and after 1.29?

What about <1.29 maps being played in 1.29?

Reply:
Do wurst inline these constants?

Reply:
I don't think wurst compiler would inline BJ constants, no, so maybe it's fine

Reply:
bj_MAX_PLAYER_SLOTS is not a constant. If it were a constant, it would get inlined, BJ or not.